### ISSUE TITLE
Document OBO Foundry decision making processes

### ISSUE THREAD

**Issue by matentzn** [OBOFoundry/OBOFoundry.github.io#2213]
I was recently approached by a member of the community who would have like to have a say in the pre-decision of to the [vote on status change requirements](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2120#issuecomment-1328883160), basically wanting to get a chance to argue that the "Pass Dashboard" criteria should be implemented in a different context, not during the status change. 

I think it would be good practice to codify decision Making in OBO Foundry.

- A decision to be made should be called in an issue and brought to the attention of OFOC
- OFOC decides if the decision is worth making. If so, they call a public vote, which must be open for at least 2 weeks (until the next OFOC call).
- During the next OFOC call, the votes are counted, the issue is closed and the decision documented in the appropriate place (with a screenshot, so no further voting is possible). 

Also we should separate decisions that need majority voting, and those than need unanimity.

- By default, all votes are majority votes. To enact the change, the change-seeking side must provide one more vote at least then the change-opposed side.
- Certain votes should be unanimous: exclusion/inclusion decisions of new ontology requests. Here, only OFOC members are allowed to vote. Here, we should decide if presence  on the OFOC call is required to cast a vote. Probably.

**Comment by nlharris** [comment]
I made a PR (https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2256) with a version of this decision-making process specific to NOR acceptance/rejection, but looking at this issue more closely, now I'm thinking maybe the SOP should have a section like this about the overall decision-making process, and the NOR decision-making process should then just be a special case of this? What do you think?

**Comment by matentzn** [comment]
Yes. This is a general "decision making" SOP, and the NOR decision making process should be documented separately (in the same doc perhaps, but different section).

**Comment by nlharris** [comment]
The PR already has a separate section about the NOR decision-making process (in the same region of the SOP as other stuff about NOR acceptance):

>## Ontology Acceptance/Rejection Decision
Once the reviewer has decided whether a new ontology meets the requirements, they present their review and recommendation to the OBO Ops committee.
The committee can ask the reviewer for clarifications, and then the people on the call make the decision about whether to accept the ontology.
The decision is made by “consensus” on the call: i.e., no one objects strenuously.
No quorum (minimum number of attendees) on the call is required.

Is that ok?

**Comment by matentzn** [comment]
I have removed the attn: OFOC call label, because as of now it is unclear what we should do in the call exactly. I also do not know to what extend this tickets content has already been incorporated by your work @nlharris.

**Comment by nlharris** [comment]
I think maybe this is done and the ticket can be closed?

**Comment by matentzn** [comment]
As far as I can see, this is done for a particular type of decisions (new ontology request decisions). What I was hoping for was a more general SOP for decision-making that is the default for _all_ decisions we have to make, including decisions about infrastructure (PURL, Dashboard etc) and OMO metadata. So this is more about general governance.

**Comment by nlharris** [comment]
Does https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/SOP.md now satisfy this request, or is more needed?

**Comment by matentzn** [comment]
I think what is needed is a paragraph called `Decision making in the OBO Foundry` that documents our general framework for decision making, with a note that specific kinds of decisions can have slightly different processes.

**Comment by nlharris** [comment]
Can someone contribute rough text for that?

**Comment by bpeters42** [comment]
Update call 6/10, @bpeters42  believes this has been worked on, and should be consolidated as there are multiple tickets.

**Comment by bpeters42** [comment]
I have not yet gone through all of the tickets related to voting, and was hoping to do that and consolidate before closing.
